### ISSUE TITLE
TFServing docs are incorrect; we no longer assign a public IP by default

### ISSUE THREAD

**Issue by jlewi** [kubeflow/kubeflow#296]
The docs:
https://github.com/kubeflow/kubeflow/blob/master/user_guide.md#serve-a-model

Incorrectly describe using an external IP for the model.

But by default we don't serve models on external IP addresses. We also don't currently have an option to easily allow that (#295).

We should update the docs and tell users

- How to use port-forward to send traffic to their pods.
- How to enable serving on an external IP address (once #295) is fixed.

**Comment by fxue** [comment]
I will work on this once PR for #295 is merged.

**Comment by jlewi** [comment]
@fxue  #295 is merged. Can we update the docs?

**Comment by jlewi** [comment]
Duplicate of #233
